src/control/server/storage/faults.go

```diff
@@ -59,16 +66,39 @@ func FaultScmInvalidPMem(msg string) *fault.Fault {
 		recreateRegionsStr)
 }
 
-// FaultScmNoModules represents an error where no PMem modules exist.
-var FaultScmNoModules = storageFault(code.ScmNoModules,
-	"No PMem modules exist on storage server",
-	"Install PMem modules and retry command")
+// FaultScmTmpfsLowMem indicates that total RAM is insufficient to support given configuration.
+func FaultScmTmpfsLowMem(memTmpfsMin, scmSize uint64) *fault.Fault {
+	return storageFault(
+		code.ScmTmpfsLowMem,
+		fmt.Sprintf("Total system memory (RAM) insufficient for tmpfs SCM, want %s have "+
+			"%s (%d bytes)", humanize.IBytes(memTmpfsMin), humanize.IBytes(scmSize),
```

Comment:
Not sure that including the raw byte count in this message is all that helpful.

Reply:
done

Reply:
Nit, when I am doing sustaining and/or performance tasks, it is quite useful to also have the size in bytes.